### PR TITLE
fixed wget OO downloads urls in fq2bam and dv1 dockers for V3.0 release

### PR DESCRIPTION
**Pull request by yuk12**
fixed wget OO downloads urls in fq2bam and dv1 dockers for V3.0 release

### CODE REVIEW

File: pipelines/deepvariant-based-germline-variant-calling-fq2vcf/Dockerfile_part1 [hunk 1]
```diff
@@ -28,7 +30,10 @@ RUN echo "source activate dv_env" >> ~/.bashrc
 RUN source ~/.bashrc
 
 WORKDIR /
-RUN git clone --recursive https://github.com/IntelLabs/Open-Omics-Acceleration-Framework.git
+#RUN git clone --recursive https://github.com/IntelLabs/Open-Omics-Acceleration-Framework.git
```
Comment (sanchit-misra) [inline pipelines/deepvariant-based-germline-variant-calling-fq2vcf/Dockerfile_part1:33]: delete this line

Comment (sanchit-misra) [reply pipelines/deepvariant-based-germline-variant-calling-fq2vcf/Dockerfile_part1:33]: In addition, I see some commented code in the Docker file part 2 and the Docker file of fq2sortedbam as well. Please delete those.

Comment (vasimuddin) [reply pipelines/deepvariant-based-germline-variant-calling-fq2vcf/Dockerfile_part1]: Done, part2 has some original comments, kept them.

**Review by sanchit-misra — APPROVED** [review]
Looks good.

**Pull request merged**